src/test/regress/expected/pg_dump.out

```diff
@@ -161,3 +161,31 @@ DETAIL:  drop cascades to table data
 drop cascades to table dist_columnar
 drop cascades to table simple_columnar
 drop cascades to table "weird.table"
+CREATE SCHEMA dumper;
+CREATE TABLE data (
+	key int,
+	value text
+);
+SELECT create_distributed_table('data', 'key');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+COPY data FROM STDIN WITH (format csv, delimiter '|', escape '\');
+-- run pg_dump on worker (which has shards)
+\COPY output FROM PROGRAM 'PGAPPNAME=pg_dump pg_dump -f results/pg_dump.tmp -h localhost -p 57637 -U postgres -d regression -n dumper --quote-all-identifiers'
```

Comment:
Just curious: why do we pass 'PGAPPNAME=pg_dump`? That seems like the default already?